src/router.js

```diff
@@ -1,25 +1,38 @@
-const express = require('express');
-const jwt = require('jsonwebtoken');
+const express = require("express");
+const jwt = require("jsonwebtoken");
 
 const router = express.Router();
 
 const mockUser = {
-    username: 'authguy',
-    password: 'mypassword',
-    profile: {
-        firstName: 'Chris',
-        lastName: 'Wolstenholme',
-        age: 43
-    }
+  username: "authguy",
+  password: "mypassword",
+  profile: {
+    firstName: "Chris",
+    lastName: "Wolstenholme",
+    age: 43,
+  },
 };
 
-router.post('/login', (req, res) => {
+router.post("/login", (req, res) => {
+  const { username, password } = req.body;
+  console.log(username, password)
+  if (username === mockUser.username && password === mockUser.password) {
+    const payload = { username: mockUser.username }
+    const token = jwt.sign(payload, process.env.JWT_SECRET);
+    res.status(200).json(token);
```

Comment:
You're missing a return here. In the case that the username and password are correct, you will first send 
```js
res.status(200).json(token)
```
then 
```js
res.status(401).json({ error: "Username or password incorrect, you idiot" });
```